containers/vsonline-linux/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,26 @@
+{
+	"name": "VS Online (Linux Default)",
+	"dockerFile": "Dockerfile",
+	"remoteUser": "vsonline",
+	"workspaceMount": "source=${localWorkspaceFolder},target=/home/vsonline/workspace,type=bind,consistency=cached",
+	"workspaceFolder": "/home/vsonline/workspace",
+	"runArgs": [ "--cap-add=SYS_PTRACE", "--security-opt", "seccomp=unconfined" ],
+
+	// Add the IDs of extensions you want installed when the container is created.
+	"extensions": [
+		"MS-vsliveshare.vsliveshare",
+		"VisualStudioExptTeam.vscodeintellicode"
+	],
```

Comment:
There are a couple other extensions we currently install. Should we include those as well?
```
"ms-vsonline.vsonline",
"GitHub.vscode-pull-request-github"
```

Reply:
Done